tests/testthat/test-dbconnect.R

```diff
@@ -13,20 +13,28 @@ for (ii in 1:length(DB_STRINGS)) {
   conn_str <- DB_STRINGS[ii]
 
   test_that(paste0("database connection works: ", name), {
+    if (name == "postgres") {
+      skip_if_not_installed("RPostgres")
+    } else {
+      skip_if_not_installed("RMariaDB")
```

Comment:
Ah thanks -- if RMariaDB isn't installed, does dbpath currently error on load? I've added an issue here (but might have gotten things wrong since my R pkg chops are rusty 😬)

https://github.com/machow/dbpath/issues/7

Reply:
It doesn't error but it prints an error message

```
Loading required namespace: RMariaDB
Failed with error:  ‘there is no package called ‘RMariaDB’’
```

Just took a look at `import_from()` and I see what's going on, also an easy fix.